openmetadata-ui/src/main/resources/ui/src/components/BotsDetail/BotsDetail.component.tsx

```diff
@@ -293,11 +293,22 @@ const BotsDetail: FC<BotsDetailProp> = ({
 
   const getBotsTokenExpiryDate = () => {
     if (botsTokenExpiry) {
+      // get the current date timestamp
+      const currentTimeStamp = moment().valueOf();
+
+      const isTokenExpired = isEqual(currentTimeStamp, botsTokenExpiry);
```

Comment:
Why can't we simply check 
```
currentTimeStamp === botsTokenExpiry
```
Is there anything additional we need from lodash?

Reply:
No, Both do the same thing. I will remove that and do a simple check there.